module/x/gravity/keeper/governance_proposals.go

```diff
@@ -179,6 +186,11 @@ func (k Keeper) HandleAirdropProposal(ctx sdk.Context, p *types.AirdropProposal)
 		}
 	}
 
+	if !totalRequiredDecCoin.Amount.Equal(totalSent) {
+		ctx.Logger().Info("Airdrop failed to excute Invalid amount sent ", totalRequiredDecCoin.Amount, " != ", totalSent)
```

Comment:
I think this log will look weird, something like the following

```
TIMESTAMP INFO Airdrop failed to excute Invalid amount sent
    12845643232 !=
    2545535353
```
I'd recommend changing this to `ctx.Logger().Info("Airdrop failed to execute: Invalid amount sent", "sent", totalSent, "expected", totalRequiredDecCoin.Amount)`